core/domain/fs_services.py

```diff
@@ -91,6 +91,38 @@ def save_original_and_compressed_versions_of_image(
             micro_image_content, mimetype=mimetype)
 
 
+def move_images_between_directories(
+        source_entity_type, source_entity_id, target_entity_type,
+        target_entity_id):
+    """Move all the images from the source entity's directory to the target
```

Comment:
This is not an operation that is typically done between actual entities. I suggest reframing as "moving from a temp directory to a main directory".

Reply:
Removed this function